README.md

```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
 ```scala
 addSbtPlugin("io.chrisdavenport" %% "sbt-npm-dependencies" % "<version>")
 ```
+
+In your `build.sbt`, enable it with `enablePlugins(NpmDependenciesPlugin)` and declare your dependencies with
+
+```scala
+Compile / npmDependencies ++= Seq(
+  "npmPackageName" -> "version",
+  "secondNpmPackageName" -> "version",
+  ...
+)
+```
+
+## Internals
+
+(For integration with other build systems)
+
+sbt-npm-dependencies will export a file called `NPM_DEPENDENCIES` containing the dependencies in the following format:
```

Comment:
Where is this file exported to?

Reply:
To the jar top-level. I don't know how to say this.

Reply:
"to the jar" is fine